v2g-liberty-cards/src/edit-charger-settings-dialog.ts

```diff
@@ -191,9 +230,7 @@ class EditChargerSettingsDialog extends DialogBase {
         evt => (this._useReducedMaxPower = evt.target.checked ? 'on' : 'off')
       )}
       ${isUsingReducedMaxPower ? this._renderReducedMaxPower() : nothing}
-      <ha-alert alert-type="info">
-        <ha-markdown .content=${info}></ha-markdown>
-      </ha-alert>
+      ${this._renderLoadbalancerInfo()}
```

Comment:
Please move the implementation of `_renderLoadBalancerInfo` down below this first call.
That way it is easier to read the code "top to bottom".

Reply:
Moving it to the utils